charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-etcd3.rules.yaml

```
@@ -65,30 +65,32 @@ groups:
       description: Etcd3 pod {{ $labels.pod }} has seen {{ $value }} proposal failures
         within the last hour.
       summary: High number of failed etcd proposals
-  
+
   - record: shoot:etcd_object_counts:sum_by_resource
     expr: sum(etcd_object_counts) by (resource)
-  
+
   # etcd backup failure alerts
   - alert: KubeEtcdDeltaBackupFailed
-    expr: changes(etcdbr_snapshot_latest_revision{job="kube-etcd3-backup-restore",role="main",kind="Incr"}[15m]) >= 1 and ON(job,role,kind) changes(etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",role="main",kind="Incr",succeeded="true"}[15m]) < 1
+    expr: (time() - etcdbr_snapshot_latest_timestamp{job="kube-etcd3-backup-restore",kind="Incr",role="main"} > bool 900) + (etcdbr_snapshot_required{kind="Incr", role="main"} == 1) == 2
```

```suggestion
    expr: (time() - etcdbr_snapshot_latest_timestamp{job="kube-etcd3-backup-restore",kind="Incr",role="main"} > bool 900) + (etcdbr_snapshot_required{kind="Incr", role="main"} >= bool 1) == 2
```

The metric snapshot_required can still emit values greater than 1, which still means snapshot is required. Only 0 values means snapshot is not required. Also, in this case, it would make sense to use `bool` to convert `>1` values to 1.

---

done